velox/exec/HashAggregation.h

```diff
@@ -72,6 +72,16 @@ class HashAggregation : public Operator {
 
   RowVectorPtr getDistinctOutput();
 
+  // Setup the projections for accessing grouping keys stored in the grouping
+  // set. 'groupingKeyInputChannels' provides the map from the key column index
+  // in the grouping set (as vector index) to key column channel in the input
+  // (as vector value). 'outputChannelProjections' provides the map from the key
+  // column channel in the output (as vector index)  to the key column index in
+  // the grouping set (as vector value).
+  void setupGroupingKeyChannelProjections(
+      std::vector<column_index_t>& groupingKeyInputChannels,
+      std::vector<column_index_t>& groupingKeyOutputChannels) const;
```

Comment:
Should we, instead of passing reference as the way of returning, considering to return a pair of vectors? Because the reference passed in is always empty.

Reply:
I might prefer to return multiple data structures as input args.